esrally/track/loader.py

```diff
@@ -83,16 +81,11 @@ def register_track_processor(self, processor):
             processor.decompressor = Decompressor()
         self.track_processors.append(processor)
 
+    @cached_property
```

Comment:
IMHO it's not ideal that this property has a side-effect. It could also lead to issues that are hard to track down, e.g. when we want to register additional track processors lazily, i.e. after we've already retrieved the `processors` property once, because then additionally registered processors won't be recognized without any indication (e.g. an exception).

How about we provide a method to ensure explicitly that the default behavior (`DefaultTrackPreparator`) is available?

Reply:
I understand the first part of this, but not 
> How about we provide a method to ensure explicitly that the default behavior (DefaultTrackPreparator) is available?

sorry.  Does changing this to `@property` circumvent the need for this suggestion? If not, I may need more explanation here or offline

Reply:
discussed offline.  will handle in my next commit